components/post-title/index.tsx

```diff
@@ -37,10 +46,6 @@ export const PostTitle = (props) => {
 	);
 };
 
-PostTitle.propTypes = {
-	tagName: PropTypes.string,
-};
-
 PostTitle.defaultProps = {
 	tagName: 'h1',
 };
```

Comment:
@Sidsector9 Should we not remove the Default Props here?

Reply:
Hey thanks for taking care of this one, I was on PTO.